src/components/VirtualKeyboard.vue

```diff
@@ -16,27 +18,42 @@ const props = defineProps<{
   heldNotes: Map<number, number>
 }>()
 
-type VirtualKey = {
+export type VirtualKey = {
```

Comment:
I don't think we've done a type export from a component before. Personally I would move this to `utils.ts` or to a new file for app-wide types, but it could be argued that having component types inside components is very Vue.